acceptance-tests/features/us31_view_case_list.feature

```diff
@@ -11,14 +11,14 @@ Acceptance Criteria:
   - Last updated date
 
 Background:
-  Given I know how many items are originally in the list
-  Given I have added values to the case_api
+  Given I view the case list
+  And I have added values to the case_api
 
-  Scenario: Case list is displayed
-    When I navigate to the case list page
-    Then check if it has more or equal the number of rows I added
-    Then check if it has the rows I added
-    Then I cleanup the case_api
+Scenario: Case list is displayed
+  When I navigate to the conveyancer frontend "/cases" page
```

Comment:
Yes the two seemingly duplicate steps were in there when I came to look at this, so I think I tried to remove this later one and the test didn't seem to work anymore. Not wanting to get to far down the rabbit whole of rework I just left this one in, but happy to go back in and have another look though if required?